modules/compiler/multi_llvm/include/multi_llvm/bitstream_reader.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) Codeplay Software Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+// Exceptions; you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#ifndef MULTI_LLVM_BITSTREAM_READER_H_INCLUDED
+#define MULTI_LLVM_BITSTREAM_READER_H_INCLUDED
+
+#include <clang/Serialization/ASTReader.h>
```

Comment:
Is multi_llvm allowed to depend on clang headers, btw?

Reply:
I don't know if there's a policy on that. To me the overhead of setting up a `multi_clang` that works just like `multi_llvm`, except for `clang`, doesn't seem to me like it would be worth it, I think it's fine like this, but happy to wait to see if other people disagree.